transcendence-app/src/main.ts

```diff
@@ -22,8 +20,6 @@ async function bootstrap() {
     .setVersion('1.0')
     .build();
   const document = SwaggerModule.createDocument(app, options);
-  const yamlString: string = yaml.stringify(document, {});
```

Comment:
We don't want anymore the spec file to regenerate when we're running the server in hot reload?

Reply:
Yes, now we generate the spec file with the swagger main to separate the concerns.